### ISSUE TITLE
IPIP Tunnel Interface not UP due to "add tunnel 'tunl0' failed: File exists"

### ISSUE THREAD

**Issue by andrewsykim** [cloudnativelabs/kube-router#273]
I've noticed that some interfaces for IPIP tunneling were being created, but not set to `UP`. I added some verbose logging and was able to get the output of the `ip tunnel` commands.

```
I0105 20:50:28.279023       1 network_routes_controller.go:791] ip tunnel args: [ip tunnel add tun-138197148207 mode ipip local 138.197.132.93 remote 138.197.148.207 dev eth0]
E0105 20:50:28.281679       1 network_routes_controller.go:793] ip command output: add tunnel "tunl0" failed: File exists
E0105 20:50:28.281705       1 network_routes_controller.go:342] Failed to inject routes due to: Route not injected for the route advertised by the node 138.197.148.207 Failed to create tunnel interface tun-138197148207. error: exit status 1
```
It seems if `tunl0` is already created, we return error since `ip tunnel` returns exit status 1. The early return happens [here](https://github.com/cloudnativelabs/kube-router/blob/master/app/controllers/network_routes_controller.go#L791-L793). This leads to nodes having all the tunnel interfaces configured, but running `netlink.LinkSetUp(link)`. 

Not familiar enough with with IPIP Tunneling to comment as to how `tunl0` is related since the IP command was `ip tunnel add tun-138197148207 mode ipip local 138.197.132.93 remote 138.197.148.207 dev eth0`. The same error happens if run the command locally.
```
# ip tunnel add tun-138197148207 mode ipip local 138.197.132.93 remote 138.197.148.207 dev eth0
add tunnel "tunl0" failed: File exists
```

**Comment by andrewsykim** [comment]
cc @murali-reddy

**Comment by andrewsykim** [comment]
After further investigation, I found out that it's actually `netlink.LinkByName` that's failing with:
```
I0105 21:20:23.209656       1 network_routes_controller.go:786] error getting link by name: numerical result out of range
```

**Comment by andrewsykim** [comment]
It seems netlink has a 15 character limit, which would explain why only _some_ of my routes are not available.

**Comment by murali-reddy** [comment]
Fixed with #274
